DigitalOffice/Services/CompanyService/Mappers/InternalUserPositionResponseMapper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Dynamic;
+using Broker.Responses;
```

Comment:
Using directives are not required by the code and can be safely removed

Reply:
fixed